### PR TITLE
nano: update Spanish translation

### PR DESCRIPTION
**Pull request by ikks**
nano: update Spanish translation

- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by renie — APPROVED** [review]
LGTM! Thank you very much for this update!

File: pages.es/common/nano.md [hunk 1]
```diff
@@ -1,28 +1,33 @@
 # nano
 
-> Editor sencillo y fácil de usar. Un clon libre y mejorado de Pico.
+> Editor sencillo y fácil de usar. Un clon libre y mejorado de `Pico`.
+> Vea también: `emacs`, `helix`, `vim`.
 > Más información: <https://nano-editor.org>.
 
-- Abre un nuevo archivo en nano:
+- Iniciar el editor:
```
Comment (kant) [inline pages.es/common/nano.md:7]:
```suggestion
- Inicia el editor:
```

File: pages.es/common/nano.md [hunk 2]
```diff
@@ -1,28 +1,33 @@
...
 
 `nano`
 
-- Abre un archivo específico:
+- Iniciar el editor sin usar archivos de configuración:
```
Comment (kant) [inline pages.es/common/nano.md:11]:
```suggestion
- Inicia el editor sin usar archivos de configuración:
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution, LGTM after the above suggestions are addressed.

**Pull request merged**